### PR TITLE
Add several misspell words

### PR DESCRIPTION
**Pull request by vikivivi**

### CODE REVIEW

File: codespell_lib/data/dictionary.txt [hunk 1]
```diff
@@ -62,8 +62,10 @@ abondon->abandon
 abondoned->abandoned
 abondoning->abandoning
 abondons->abandons
+aboout->about
 abord->abort
 aborigene->aborigine
+aborte->aborted
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt:68]: Also suggest abort.

File: codespell_lib/data/dictionary.txt [hunk 2]
```diff
@@ -2242,6 +2244,10 @@ asistance->assistance
 aske->ask
 askes->asks
 aslo->also
+asnwer->answer
+asnwered->answered
+asnwering->answering
+asnwers->answers
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt:2250]: You could also add the -er version of this typo.

File: codespell_lib/data/dictionary.txt [hunk 3]
```diff
@@ -2260,6 +2266,7 @@ assasins->assassins
 assassintation->assassination
 asscciated->associated
 assciated->associated
+asscii->ascii
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt:2269]: Please can you capitalise the correction to `ASCII`

File: codespell_lib/data/dictionary.txt [hunk 4]
```diff
@@ -4620,6 +4645,8 @@ cicruit->circuit
 cicruits->circuits
 cicular->circular
 ciculars->circulars
+cilent->client
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt:4648]: Also suggest `silent`

File: codespell_lib/data/dictionary.txt [hunk 5]
```diff
@@ -4620,6 +4645,8 @@ cicruit->circuit
...
+cilents->clients
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt:4649]: Also suggest `silents` and `silence`

File: codespell_lib/data/dictionary.txt [hunk 6]
```diff
@@ -4620,6 +4645,8 @@ cicruit->circuit
...
 cilincer->cylinder
 cilincers->cylinders
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt:4651]: I know it's not your change, but would you mind adding `silencer` and `silencers` to these too please?

File: codespell_lib/data/dictionary.txt [hunk 7]
```diff
@@ -22740,6 +22776,7 @@ scemes->schemes, scenes,
 scenarion->scenario
 scenarions->scenarios
 scence->scene
+scences->scenes
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt:22779]: Can you also suggest `science(s)` for these two.

Comment (peternewman) [inline codespell_lib/data/dictionary.txt:22779]: Probably also census for the latter.

File: codespell_lib/data/dictionary.txt [hunk 8]
```diff
@@ -23550,6 +23588,8 @@ sintakts->syntax
 sintax->syntax
 Sionist->Zionist
 Sionists->Zionists
+sirectories->directories
+sirectory->directory
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt:23592]: Can you also add separate typos of `sirector`, `sirectors` and `sirecting`

**Comment by vikivivi** [comment]
Done. In additional, I added sirect->direct  (ed, s)

**Comment by c72578** [comment]
What about `sirection`, `sirections`?

File: codespell_lib/data/dictionary.txt [hunk 9]
```diff
@@ -22739,7 +22777,8 @@ sceme->scheme, scene,
 scemes->schemes, scenes,
 scenarion->scenario
 scenarions->scenarios
-scence->scene
+scence->scene, science,
+scences->scenes, sciences, census,
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt:22781]: Can you also suggest `sense(s)` a potential phonetic typo.

**Pull request merged**